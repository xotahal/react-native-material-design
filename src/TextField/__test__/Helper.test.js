import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import ThemeProvider from '../../styles/ThemeProvider.react';

import Helper from '../Helper.react';

/* eslint-env jest */

const text = 'helper';

it('renders helper', () => {
    const helper = renderer
        .create(
            <ThemeProvider>
                <Helper>{text}</Helper>
            </ThemeProvider>)
        .toJSON();

    expect(helper)
        .toMatchSnapshot();
});
