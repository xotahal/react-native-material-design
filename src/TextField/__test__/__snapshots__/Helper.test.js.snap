// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders helper 1`] = `
<View
  style={
    Object {
      "bottom": 0,
      "left": 0,
      "paddingVertical": 4,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    collapsable={undefined}
    ellipsizeMode="tail"
    numberOfLines={1}
    style={
      Object {
        "backgroundColor": "transparent",
        "color": "rgba(0, 0, 0, .38)",
        "fontSize": 12,
      }
    }
  >
    helper
  </Text>
</View>
`;
